buildSrc/atlasPlugin.gradle

```diff
@@ -0,0 +1,49 @@
+
+sourceSets {
+    generated
+}
+
+idea {
+    module {
+        sourceDirs += file('src/generated/java')
+        generatedSourceDirs += file('src/generated/java')
+    }
+}
+
+configurations {
+    generatedCompile.extendsFrom compile
+}
+
+task cleanSchema(type: Delete) {
+    delete 'src/generated/java'
+}
+
+task generateSchema(type: JavaExec) {
+   main = 'com.palantir.salt.atlas.SaltSchema'
```

Comment:
Can we auto-detect these, e.g. `com.palantir.example.profile.schema.ProfileSchema`?